content/id/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Belajar tentang Pod Kubernetes.</li>
+                    <li>Belajar tentang Node Kubernetes.</li>
+                    <li>Mengatasi masalah pada aplikasi-aplikasi yang di-<i>deploy</i>.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Pod Kubernetes</h2>
+                <p>Ketika kamu membuat sebuah Deployment di Modul <a href="/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes membuatkan sebuah <b>Pod</b> untuk menempatkan instans aplikasimu. Pod merupakan sebuah abstraksi Kubernetes yang merepresentasikan sebuah grup yang terdiri dari satu atau lebih kontainer (seperti Docker), dan beberapa sumber daya bersama untuk kontainer-kontainer itu. Sumber daya tersebut termasuk:</p>
+                <ul>
+                    <li>Penyimpanan bersama, disebut Volume</li>
+                    <li>Jaringan, sebagai satu alamat IP klaster unik</li>
+                    <li>Informasi tentang bagaimana mejalankan tiap kontainer, seperti versi <i>image</i> kontainer atau porta spesifik yang digunakan</li>
+                </ul>
+                <p>Pod memodelkan sebuah "logical host" spesifik aplikasi dan dapat berisi beberapa kontainer aplikasi berbeda yang relatif terkait erat. Contohnya, sebuah Pod mungkin terdiri atas kontainer aplikasi Node.js dan juga kontainer berbeda yang bertugas menyediakan data untuk dipublikasikan ke server web Node.js. Kontainer-kontainer dalam sebuah Pod berbagi satu alamat IP dan ruang porta, selalu terletak bersama dan terjadwal bersama, dan berjalankan dalam satu konteks bersama (<i>shared context</i>) pada Node yang sama.</p>
+
+            <p>Pod merupakan unit terkecil dalam platform Kubernetes. Ketika kita membuat sebuat Deployment, Deployment tersebut membuat Pod dengan kontainer-kontainer di dalamnya (bukannya dengan membuat kontainer secara langsung). Tiap Pod terikat langsung dengan Node di mana dia dijadwalkan dan tetap di sana sampai terminasi (berdasarkan <i>restart policy</i>) atau pengapusan. Jika terjadi kegagalan pada sebuah Node, Pod indentik akan dijadwalkan di Node lain dalam klaster.</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Ringkasan:</h3>
+                    <ul>
+                        <li>Pod</li>
+                        <li>Node</li>
+                        <li>Perintah utama kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                        <p><i>
+                            Pod merupakan sebuah grup yang terdiri dari satu atau lebih kontainer aplikasi (seperti Docker) dan berisi penyimpanan bersama (volume), alamat IP dan informasi tentang bagaimana menjalankan mereka.
```

Comment:
Container is API name, no need to translate. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```suggestion
                            Pod merupakan sebuah grup yang terdiri dari satu atau lebih Container aplikasi (seperti Docker) dan berisi penyimpanan bersama (_volume_), alamat IP dan informasi tentang bagaimana menjalankan mereka.
```